lib/public/views/Tags/format/formatTagsList.js

```diff
@@ -26,12 +26,13 @@ export const formatTagsList = (tags, configuration) => {
     const { flex } = configuration || {};
 
     /**
-     * Wrapp the tag badge component to remove the description
+     * Wrap the tag badge component to include the description
      *
      * @param {tag} tag the tag to display
      * @return {Component} the badge component
      */
-    const tagToBadge = ({ text, archived, color }) => tagBadge({ text, archived, color });
+    const tagToBadge = ({ text, archived, color, description }) =>
+        tagBadge({ text, archived, color, description }); // Pass description to tagBadge
```

Comment:
I suppose then that the function simply makes no sense, no? From what I see you can directly use `tagBadge`